src/core/lib/surface/completion_queue.c

```diff
@@ -592,6 +598,7 @@ grpc_event grpc_completion_queue_next(grpc_completion_queue *cc,
       gpr_mu_lock(cc->mu);
       continue;
     } else {
+      gpr_atm_no_barrier_fetch_add(&cc->num_polls, 1);
```

Comment:
Suggest making this a normal int (no need for an atomic here) and just locking when we do the get